### PR TITLE
chore(agw): clean up grpc vs net.listener network scheme handling

### PR DESCRIPTION
**Pull request by zbrock**
* clean up grpc vs net.listener network scheme handling
* clean up repeated grpc server setup code
* Fix missing gofmt run on script file

## Summary

Per PR comments on 10158

## Test Plan
Ensured that gRPC service still comes up on local box.
Unit tests should be sufficient.

## Additional Information

N/A

### CODE REVIEW

File: src/go/agwd/config/config_test.go [hunk 1]
```diff
@@ -132,6 +132,53 @@ func TestParseTarget(t *testing.T) {
...
+			},
+		},
+		{
+			input: resolver.Target{
+				Scheme: "ipv6",
+			},
+			want: resolver.Target{
+				Scheme: "tcp6",
+			},
+		},
```
Comment (mstre123) [inline src/go/agwd/config/config_test.go:166]: probably good to add one more test that scheme with tcp4 outputs scheme with tcp4

File: src/go/agwd/config/config.go [hunk 2]
```diff
@@ -53,24 +53,35 @@ const (
...
 		}
 	}
 	return grpcutil.ParseTarget(target, false)
 }
 
+// TargetSchemeAdapter takes a gRPC formatted Target such as {ipv4, ipv6}
+// and translates to net.Listen's required network type such as {tcp4, tcp6}.
+func TargetSchemeAdapter(target resolver.Target) resolver.Target {
+	if target.Scheme == ipv4Scheme {
+		target.Scheme = tcp4Scheme
```
Comment (mstre123) [inline src/go/agwd/config/config.go:77]: should we do immediate return in the conditional statements like we do in ParseTarget - makes it clearer that the target.Scheme isn't being altered by anything further down in the function.

**Review by mstre123 — APPROVED** [review]
lgtm

File: src/go/agwd/config/config.go [hunk 3]
```diff
@@ -53,24 +53,35 @@ const (
...
+			Scheme:   ipv6Scheme,
 			Endpoint: target[len(ipv6Scheme)+1:],
 		}
 	}
 	return grpcutil.ParseTarget(target, false)
 }
 
+// TargetSchemeAdapter takes a gRPC formatted Target such as {ipv4, ipv6}
+// and translates to net.Listen's required network type such as {tcp4, tcp6}.
+func TargetSchemeAdapter(target resolver.Target) resolver.Target {
```
Comment (markjen) [inline src/go/agwd/config/config.go:75]: interesting. why not `func NewTargetListner(target resolver.Target) (net.Listner, error)`

Comment (zbrock) [reply src/go/agwd/config/config.go:75]: I started adding an injectable Factory interface for net.Listeners so I could write tests for it, but the code ended up being mostly abstractions to make it testable with almost no actual business logic. This was simpler to get under test coverage.

**Pull request closed without merging**